deploy/bin/build

```diff
@@ -123,6 +138,7 @@ esac
 if [ -f packer-artifacts.json ]; then
   jq '
     .builds[]
+    | select(.builder_type == "amazon-ebs")
```

Comment:
I don't know what I had in mind what I thought using `jq` would be a good idea.

Reply:
It works fine, it just doesn't fit in my brain :)